financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkinglinkloginwarmup/NetworkingLinkLoginWarmupScreen.kt

```diff
@@ -84,181 +88,165 @@ private fun NetworkingLinkLoginWarmupContent(
-                    scrollState = scrollState,
-                    payload = payload(),
-                    onClickableTextClick = onClickableTextClick,
-                    onContinueClick = onContinueClick
-                )
-
-                is Fail -> UnclassifiedErrorContent(
-                    error = disableNetworking.error,
-                    onCloseFromErrorClick = onCloseFromErrorClick
-                )
-            }
+            Uninitialized,
+            is Loading -> FullScreenGenericLoading()
 
             is Fail -> UnclassifiedErrorContent(
                 error = payload.error,
                 onCloseFromErrorClick = onCloseFromErrorClick
             )
-        }
-    }
-}
 
-@Composable
-private fun NetworkingLinkLoginWarmupLoading() {
-    Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
-        CircularProgressIndicator(
-            color = FinancialConnectionsTheme.colors.iconBrand
-        )
+            is Success -> NetworkingLinkLoginWarmupLoaded(
+                lazyListState = lazyListState,
+                payload = payload(),
+                disableNetworkingAsync = state.disableNetworkingAsync,
+                onSkipClicked = onSkipClicked,
+                onCloseFromErrorClick = onCloseFromErrorClick,
+                onContinueClick = onContinueClick
+            )
+        }
     }
 }
 
+@OptIn(ExperimentalComposeUiApi::class)
 @Composable
 private fun NetworkingLinkLoginWarmupLoaded(
-    scrollState: ScrollState,
+    lazyListState: LazyListState,
     payload: Payload,
-    onClickableTextClick: (String) -> Unit,
+    disableNetworkingAsync: Async<FinancialConnectionsSessionManifest>,
+    onCloseFromErrorClick: (Throwable) -> Unit,
+    onSkipClicked: () -> Unit,
     onContinueClick: () -> Unit,
 ) {
-    Column(
-        modifier = Modifier
-            .fillMaxSize()
-            .verticalScroll(scrollState)
-            .padding(
-                top = 0.dp,
-                start = 24.dp,
-                end = 24.dp,
-                bottom = 24.dp
-            )
-    ) {
-        Spacer(modifier = Modifier.size(16.dp))
-        Title(
-            onClickableTextClick = onClickableTextClick
+    if (disableNetworkingAsync is Fail) {
+        UnclassifiedErrorContent(
+            error = disableNetworkingAsync.error,
+            onCloseFromErrorClick = onCloseFromErrorClick
         )
-        Spacer(modifier = Modifier.size(8.dp))
-        Description(onClickableTextClick)
-        Spacer(modifier = Modifier.size(24.dp))
-        ExistingEmailSection(
-            email = payload.email,
-            onContinueClick = onContinueClick
+    } else {
+        Layout(
+            lazyListState = lazyListState,
+            body = {
+                item {
+                    CircleIcon(painterResource(id = R.drawable.stripe_ic_person))
+                    Spacer(modifier = Modifier.size(16.dp))
```

Comment:
Do we need to wrap the content in a `Column` here? If yes, we could also use `Arrangement.spacedBy` instead of the spacers.